src/auth/user-record.ts

```diff
@@ -41,6 +42,14 @@ function parseDate(time: any): string | null {
   return null;
 }
 
+interface SecondFactor {
```

Comment:
The public API uses `UpdateMultiFactorInfoRequest` and `CreateMultiFactorInfoRequest` instead of this. Can we use the same here for consistency?

Reply:
Done.